doc/source/examples/multiphysics/laser-meltpool/laser-meltpool.rst

```diff
@@ -91,36 +95,36 @@ All the four boundary conditions are ``noslip``, and the heat transfer boundary
         subsection bc 0
         	set id = 0
     	set type	      = convection-radiation
-            set h	      	      = 15
-            set Tinf	      = 20
-            set emissivity        = 0.5
+            set h                   = 80
+            set Tinf                = 20
+            set emissivity          = 0.6
         end
         subsection bc 1
         	set id = 1
     	set type	      = convection-radiation
-            set h	              = 15
-            set Tinf	      = 20
-            set emissivity        = 0.5
+            set h                   = 80
+            set Tinf                = 20
+            set emissivity          = 0.6
         end
         subsection bc 2
         	set id = 2
     	set type	      = convection-radiation
-            set h	              = 15
-            set Tinf	      = 20
-            set emissivity        = 0.5
+            set h                   = 80
+            set Tinf                = 20
+            set emissivity          = 0.6
         end
         subsection bc 3
         	set id = 3
     	set type	      = convection-radiation
-            set h	              = 15
-            set Tinf	      = 20
-            set emissivity        = 0.5
+            set h                   = 80
+            set Tinf                = 20
+            set emissivity          = 0.6
         end
     end
 
 
 The ``multiphysics`` subsection enables to turn on (``true``) 
-and off (``false``) the physics of interest. Here ``heat transfer`` is enabled.
+and off (``false``) the physics of interest. Here ``heat transfer``, ``buoyancy force``, ``fluid dynamics``, ``VOF`` are enabled. Note that ``VOF`` is not effective in the current version of the exmple, as the whole domain is filled with fluid 0, but it will be functional in the updated versions of this example.
```

Comment:
I would make a second example with VOF instead of trying to make one mega example. So I would disable vOF.